robottelo/host_helpers/satellite_mixins.py

```diff
@@ -443,3 +443,45 @@ def api_factory(self):
     @lru_cache
     def ui_factory(self, session):
         return UIFactory(self, session=session)
+
+
+class IoPSetup:
+    """Helper for configuring on prem Insights Advisor engine."""
+
+    iop_settings = settings.rh_cloud.iop_advisor_engine
+
+    def podman_login(self, username=None, token=None, registry=None):
```

Comment:
non-blocking: I'd see this as a ContentHost method or a method in some ContentHost mixin.